list.js

```diff
@@ -0,0 +1,61 @@
+class Component {
+  constructor() {
+    this.container = document.getElementById("ingredients-list");
```

Comment:
```suggestion
    this.ref = document.getElementById("ingredientsList");
```
we call it `ref` in other places and we use camelCase for classnames

Reply:
this is still to be fixed